01week/datatypes.js

```diff
@@ -67,4 +67,5 @@ var b = -3;
 
 console.log (a + b == -4);
 
-console.log (a + b == 76)
\ No newline at end of file
+console.log (a + b == 76)
```

Comment:
If you want full credit please work through each of the instructions:
https://aca-8wk-js211.herokuapp.com/_book/01Week/01DayClass.html#part-one-instructions